package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "brewedreads.com",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "prisma generate && next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "prisma": {
+    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
+  },
+  "dependencies": {
+    "@prisma/client": "^5.5.2",
```

Comment:
please also install

- react-query for fetching data
- react-hook-form
-

Reply:
install next-auth

Reply:
Also added, it will be in the next commit.